web/models/activity_log.ex

```diff
@@ -147,6 +151,21 @@ defmodule Ask.ActivityLog do
     })
   end
 
+  def rename_folder(project, conn, folder, old_folder_name, new_folder_name) do
+    create("rename", project, conn, folder, %{
+      old_folder_name: old_folder_name,
+      new_folder_name: new_folder_name
+    })
+  end
+
+  def change_folder(project, conn, survey, old_folder_name, new_folder_name) do
+    create("change_folder", project, conn, survey, %{
+      survey_name: survey.name,
+      old_folder_name: old_folder_name,
+      new_folder_name: to_string(new_folder_name)
```

Comment:
Why is `to_string` required here?

Reply:
Do you remember something about that, @hdf1986? I removed `to_string` from the expression and everything seems to be working properly.